### PR TITLE
Update linux gaming

### PR DESCRIPTION
**Pull request by ahoneybun**
This updates screenshots, text and other items for 22.04.

### CODE REVIEW

File: content/linux-gaming.md [hunk 1]
```diff
@@ -28,13 +29,13 @@ Open the <u>Pop!_Shop</u> application then either search for Steam or by clickin
 
 ![Pop!_Shop Steam](/images/linux-gaming/pop-shop_steam.png)
 
-Once installed, use the Activities Overview to search for and run <u>Steam</u>.
+Once installed, use the Application button to search for and run <u>Steam</u>.
```
Comment (leviport) [inline content/linux-gaming.md:32]: The button says "Applications", so I'm thinking Application should probably be plural to match the button, right?

File: content/linux-gaming.md [hunk 2]
```diff
@@ -60,6 +61,45 @@ Steam -> Settings -> Steam Play -> Check the box for "Enable Steam Play for supp
 
 ![Steam Settings](/images/linux-gaming/enable-steam-play_proton.png)
 
+## Using Steam
+
+Once installed, use the Application button to search for and run <u>Steam</u>.
```
Comment (leviport) [inline content/linux-gaming.md:66]: The button says "Applications", so I'm thinking Application should probably be plural to match the button, right?

File: content/linux-gaming.md [hunk 3]
```diff
@@ -155,7 +193,7 @@ Open the <u>Pop!_Shop</u> application then either search for GameHub, or navigat
 
 ![Pop!_Shop GameHub](/images/linux-gaming/pop-shop_gamehub.png)
 
-Once installed, use the Activities Overview to search for and run <u>GameHub</u>.
+Once installed, use the Application button to search for and run <u>GameHub</u>.
```
Comment (leviport) [inline content/linux-gaming.md:196]: The button says "Applications", so I'm thinking Application should probably be plural to match the button, right?

**Review by leviport — CHANGES_REQUESTED** [review]
Looks like a good update, but I have one nitpick

**Review by leviport — APPROVED** [review]

**Review by garrettjwilke — APPROVED** [review]

**Pull request merged**